rfd/0002-code-of-conduct.md

```diff
@@ -0,0 +1,266 @@
+---
+authors: JP Bruins Slot <jpbruinsslot@gmail.com>
+state: discussion
+discussion: https://github.com/minvws/nl-kat-coordination/pull/3425
+labels: process
+---
+
+# RFD 0002: Code of Conduct: Code Review
+
+## Introduction
+
+Code reviews are an essential part of the software development process,
+ensuring that the codebase remains clean, efficient, and maintainable while
+fostering collaboration and knowledge sharing among team members. For both
+authors and reviewers, approaching code reviews with the right mindset and
+strategies can significantly improve the quality of the code and the efficiency
+of the review process.
+
+By adhering to these best practices, both authors and reviewers can contribute
+to a more effective and enjoyable code review process, ultimately leading to
+higher-quality software and stronger team dynamics.
+
+Having a code of conduct for code reviews can help set expectations, establish
+a positive and respectful environment, and ensure that the review process is
+productive and beneficial for all team members.[^2]
+
+## Authors
+
+1. **Mind your reviewer**
+
+   Make choices or options that minimize time and cognitive load for the reviewer,
+   such as opting for multiple short changes instead of one massive one.
+
+2. **Satisfy preconditions**
+
+   Ensure that your code is ready for review before you send it, and if you feel that it is not entirely there yet please mark it as 'draft' to signal this to potential reviewers:
+
+   - It should work and is able to run,
+   - There should be no unresolved merge conflicts,
+   - Have adequate testing that pass,
+   - And respect the style and coding guidelines.
```

Comment:
Read the diff before revieing. Remove anything that isn't supposed to be there such as formatting only changes.